cluster/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted-v1
+  annotations:
+    kubernetes.io/description: 'This policy demonstrates best practices for
+      running Pods with minimal security priviliges.'
+    # To use custom seccomp or apparmor profiles, make a copy of this policy and
+    # add the custom profiles here.
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
```

Comment:
remember that order of PSP's isn't well defined when multiple match yet...  that becomes an issue when some apply defaulting

Reply:
Ack. Added a warning. Since this is "documentation", I think it's OK here.